public/Sjofn/homework1.html

```diff
@@ -0,0 +1,72 @@
+<html lang="en">
+<head>
+    <title>Homework 1</title>
+    <style>
+        div {
+            background-color: dodgerblue;
+        }
+        #myId{
+            color: tomato;
+            background-color: cadetblue;
+            size: A3;
```

Comment:
Doesn't look like this property really affects anything on the page. 
If you want to change size of font, better use `font-size` property.